<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressPanel</class>
 <widget class="QDockWidget" name="AddressPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>196</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>AddressPanel</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="JTextField" name="edAddress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="dispatchButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Dispatch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="releaseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Release</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="progButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Program</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="RosterEntrySelectorPanel" name="rosterBox" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JTextField</class>
   <extends>QLineEdit</extends>
   <header>jtextfield.h</header>
  </customwidget>
  <customwidget>
   <class>RosterEntrySelectorPanel</class>
   <extends>QWidget</extends>
   <header>rosterentryselectorpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
